packages/server/routes/api/events.js

```diff
@@ -281,6 +281,45 @@ router.get(
   }
 );
 
+router.put(
+  "/:eventId/status",
+  AuthController.authenticate,
+  eventIdValidator,
+  async (request, response) => {
+    const result = validationResult(request);
+
+    if (!result.isEmpty()) {
+      response.status(400).json({
+        status: "fail",
+        data: result.array(),
+      });
+      return;
+    }
+
+    const { userId, status } = request.body;
+    const { eventId } = request.params;
+
+    try {
+      const updatedEventAttendeeStatus =
+        await EventController.updateEventAttendeeStatus(
+          userId,
+          eventId,
+          status
+        );
+
+      response.status(200).json({
+        status: "success",
+        data: { updatedEventAttendeeStatus: updatedEventAttendeeStatus },
```

Comment:
`updatedEventAttendeeStatus` is a bit too long of a name for the response data. The data we're responding to the user with here is the event attendee info so just `eventAttendee` is a shorter but specific enough name to be clear to the frontend.

```js
data: {
  eventAttendee: updatedEventAttendeeStatus
}
```